### PR TITLE
Style/mvp design tweaks

### PR DESCRIPTION
**Pull request by 0xSaga**
Screen Shot 2022-01-05 at 7 33 14 PM
Screen Shot 2022-01-05 at 7 35 04 PM

### CODE REVIEW

File: packages/app/src/components/core/Link.tsx [hunk 1]
```diff
@@ -2,15 +2,37 @@
 import React from 'react';
 import { Link as RLink, NavLink as RNavLink } from 'react-router-dom';
 import { Themed } from 'theme-ui';
+import { Button, Row, Text } from './core';
+import { Link as ThemeLink } from '@theme-ui/components';
 
 export const Link = ({ children, ...props }): JSX.Element => (
```
Comment (0x0scion) [inline packages/app/src/components/core/Link.tsx:8]: rename to RouterLink

File: packages/app/src/components/core/Link.tsx [hunk 2]
```diff
@@ -2,15 +2,37 @@
...
-  <Themed.a as={RLink} {...props}>
+  <Themed.a as={RLink} {...props} sx={{ variant: 'a.primary', ...props.sx }}>
     {children}
   </Themed.a>
 );
 
 export const NavLink = ({ children, ...props }): JSX.Element => (
```
Comment (0x0scion) [inline packages/app/src/components/core/Link.tsx:14]: RouterNavLink

File: packages/app/src/components/nav/Header.tsx [hunk 3]
```diff
@@ -1,36 +1,50 @@
...
-              </NavLink>
-            )}
+            <Box>
+              {!mobile && (
+                <Row>
+                  <NavLink exact to="/">
+                    Vaults
+                  </NavLink>
+                  <Box mr={4} />
+                  <Link
```
Comment (0x0scion) [inline packages/app/src/components/nav/Header.tsx:27]: Should we use ExternalLink here?

File: packages/app/src/components/nav/Header.tsx [hunk 4]
```diff
@@ -1,36 +1,50 @@
...
+              )}
+            </Box>
           </Row>
           <Row sx={{ alignItems: 'center' }}>
             {!mobile && <Dimmer />}
             <Box sx={{ mr: 3 }} />
-            <ConnectWallet />
-            {!!mobile && <OpenMenu />}
+            {!mobile && <ConnectWallet />}
+            {(modal || !!mobile) && <ToggleMenu />}
```
Comment (0x0scion) [inline packages/app/src/components/nav/Header.tsx:43]: `modal || !!mobile` how is this different form just `!!mobile`

**Review by 0x0scion — APPROVED** [review]

File: packages/app/src/components/nav/Header.tsx (hunk 4 above)
Comment (0xSaga) [reply packages/app/src/components/nav/Header.tsx:43]: in the edgecase that the user opens the modal and then switches to larger screen size, we still want them to be able to exit that modal

Comment (0x0scion) [reply packages/app/src/components/nav/Header.tsx:50]: ooh, i though they both said modal

**Pull request merged**